testplan/runners/pools/child.py

```diff
@@ -194,13 +179,14 @@ def worker_loop(self):
             return
 
         with self._child_pool():
-            if self._pool_cfg.worker_heartbeat:
-                self.heartbeat_setup()
             message = Message(**self.metadata)
             next_possible_request = time.time()
+            next_heartbeat = time.time()
```

Comment:
here, set next_heartbeat = 0 to avoid unnecessary computation.

Reply:
It does trigger heartbeat in the first iteration. I'm not a great fan of initializing next_heartbeat to 0 though.